requirements.txt

```diff
@@ -1,11 +1,121 @@
+argon2-cffi==21.3.0
```

Comment:
Hi @Tom-Nagy , I know i've harked on about this before....but this requirements.txt file is absolutely massive? I'm not sure for the dependencies we are using that it needs to be this big. Can this be confirmed please before we review this further?